indico/web/client/js/legacy/libs/timetable/Draw.js

```diff
@@ -255,9 +232,12 @@ type(
           self._formatConveners(this.eventData.conveners)
         );
 
-        this.div.append(this.convenerDiv);
-        this.div.append(this.timeDiv);
-        this.div.append(this.locationDiv);
+        if (this.eventData.conveners) {
+          this.headerBlock.append(this.convenerDiv);
+        }
+        this.footerBlock.append(this.locationDiv);
```

Comment:
`jQuery.append` allows you to pass an array of nodes to append `this.footerBlock.append([this.locationDiv, this.timeDiv])`

Reply:
I would like to avoid using jQuery here as much as possible since this is just plain JS.
Considering it was just two divs, I kept it as two calls, but I can always do:
`[this.locationDiv, this.timeDiv].forEach(x => this.footerBlock.append(x))`

Reply:
I don't think that change would improve readability, and it's extremely legacy code anyway...

Reply:
Ah, nvm.. I thought that `this.footerBlock` is a `jQuery` object which then allows to pass an array to the `append` method.

Reply:
`append` is just a method that calls `Node.appendChild` internally.